### DISCUSSION TITLE
Help, I can't update my game.

### DISCUSSION THREAD

**Discussion by yourfather012** [Monika-After-Story/MonikaModDev#9778] [general]
Hello :). So, I'm pretty sure the latest update for Monika after story was a couple months ago (feel free to correct me if I'm wrong). But every time I try to update it, it freezes. Does anyone know how to fix this?

**Comment by Wingdinggaster656** [comment]
The reason can be various, like you have a bad internet connection, or something else.

You could consider to manually download the new version.
